src/component/Navbar/Navbar.js

```diff
@@ -1,13 +1,49 @@
-import React from 'react'
-import { Link } from 'react-router-dom';
+import React from "react";
+import "../../App.css";
+import { Link } from "react-router-dom";
+// import { } from "../Footer/Footer";
+import { FaHome, FaList, FaCompass , FaThumbsUp , FaHistory , FaClock } from "react-icons/fa";
 
 function Navbar() {
-    return (
-        <div>
-            <Link to="/">Home</Link>
-            <Link to="/playlist">Playlistpage</Link>
-            <Link to="/videolist">Videolistpage</Link>
+  return (
+    <div className="navbar maple-flex">
+      <Link to="/">
+        <div className="nav-pill maple-flex">
+          <FaHome />
+          <h1 className="txt-m">Home</h1>
         </div>
-    )
+      </Link>
+      <Link to="/videolist">
```

Comment:
you can easily extract this part as an array and map it through. Will make your code cleaner and your JSX component lighter. Also, you wont need to add link tags again and again

Reply:
Thanks for the suggestion Piyush extracted the Navbar items